static/app/views/alerts/issueRuleEditor/sentryAppRuleModal.tsx

```diff
@@ -0,0 +1,39 @@
+import {Component, Fragment} from 'react';
+
+import {ModalRenderProps} from 'app/actionCreators/modal';
+import {tct} from 'app/locale';
+import SentryAppExternalForm, {
+  Config,
+} from 'app/views/organizationIntegrations/sentryAppExternalForm';
+
+type Props = ModalRenderProps & {
+  sentryAppInstallationUuid: string;
+  appName: string;
+  config: Config;
+  action: 'create' | 'update';
+  onSubmitSuccess: (...params: any[]) => void;
```

Comment:
Can we do better with the types here?

Reply:
`Function` type is probably worse. Really I'm wondering if we can define the parameters.

Maybe it can be this actually

```tsx
type Props = ModalRenderProps 
	& Pick<
		React.ComponentProps<SentryAppExternalForm>, 
		'sentryAppInstallationUuid' | 'appName' | 'config' | 'action' | 'onSubmitSuccess'
	>;
```

Reply:
I see the types for `onSubmitSuccess` in the `SentryAppExternalForm` are already not great haha